src/nls/messages_en.json

```diff
@@ -66,6 +66,17 @@
     "dashboard.taskRun.status.running": "Running",
     "dashboard.taskRun.status.succeeded": "Completed",
     "dashboard.taskRun.status.waiting": "Waiting",
+    "dashboard.triggerBinding.dateCreated": "Date Created:",
+    "dashboard.triggerBinding.details": "Details",
+    "dashboard.triggerBinding.errorloading": "Error loading TriggerBinding",
+    "dashboard.triggerBinding.labels": "Labels:",
+    "dashboard.triggerBinding.namespace": "Namespace:",
+    "dashboard.triggerBinding.noLabels": "None",
+    "dashboard.triggerBinding.noParams": "No parameters found for this binding.",
+    "dashboard.triggerBinding.title": "Parameters",
+    "dashboard.triggerBinding.unavailable": "TriggerBinding not available",
+    "dashboard.triggerBinding.value": "Value",
+    "dashboard.triggerBinding.yaml": "Yaml",
```

Comment:
no need to translate `YAML`, and it should be capitalised in use as it's an acronym